demos/node-classification/gcn/gcn-cora-node-classification-example.ipynb

```diff
@@ -4,14 +4,36 @@
    "cell_type": "markdown",
```

Comment:
<p>We can remove this extra step as I suggested above.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stellargraph/stellargraph/pull/1125/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='42'/>